app.js

```diff
@@ -36,4 +37,237 @@ form.addEventListener('submit', (e) => {
     document.getElementById('email-address').style.color = 'grey';
     errorElement.innerText = '';
   }
-});
\ No newline at end of file
+
+  // JS for preserve data in the browsererve Data
+  const userInput = {
+    storeName: name.value,
+    storeEmail: email.value,
+    storeMessage: message.value,
+  };
+  localStorage.setItem('userInput', JSON.stringify(userInput));
+
+  // window.onload = function () {
+  //   const data = JSON.parse(localStorage.getItem('userInput'));
+  //   if (data) {
+  //     name.value =atus data.storeName;
+  //     email.value = data.storeEmail;
+  //     message.value = data.storeMessage;
+  //   }
+  // };
+});
```

Comment:
- [ ] Kindly consider deleting the commented code in your codebase. This is because the codes commented make your code look messy and therefore very hard to maintain 😞.